src/compression_wrapper.c

```diff
@@ -255,6 +277,8 @@ cr_compression_type(const char *name)
         type = CR_CW_XZ_COMPRESSION;
     if (!g_strcmp0(name_lower, "zck"))
         type = CR_CW_ZCK_COMPRESSION;
+    if (!g_strcmp0(name_lower, "zst") || !g_strcmp0(name_lower, "zstd"))
```

Comment:
What's this "zst" and "zstd" choice? The compression is named "Zstandard" and "zstd". I think you are still mistaken it with "zst" file name extension. Can you remove "zst" option from here?

Reply:
yes, sorry, I went through it again, hopefully "zst" is just an extension now.